runtime/rococo/src/xcm_config.rs

```diff
@@ -389,3 +399,73 @@ impl pallet_xcm::Config for Runtime {
 	type ReachableDest = ReachableDest;
 	type AdminOrigin = EnsureRoot<AccountId>;
 }
+
+/// Constant derivation function for Tinkernet Multisigs.
+/// Uses the Tinkernet genesis hash as a salt.
+fn derive_tinkernet_multisig(id: u128) -> Result<AccountId, <u32 as TryFrom<u128>>::Error> {
```

Comment:
Instead of putting the logic here and repeating it on the XCM config on Kusama, let's put all of this under `xcm/xcm-builder` so that the logic is shared between the two (and upcoming) runtimes.

Reply:
Will do!

Reply:
Done!